test/modellib/ModelHelper.hpp

```diff
@@ -12,13 +12,19 @@
 //
 //===----------------------------------------------------------------------===//
 
+#pragma once
```

Comment:
Need to add a header guard

Reply:
file was removed because not in use anymore